src/components/Tour/Tour.scss

```
@@ -0,0 +1,70 @@
+
+.tour-scrim {
+  position: fixed;
+  inset: 0;
+  background: rgba(0, 0, 0, 0.6);
+  z-index: 1800;
+}
+
+.tour-popover {
+  position: fixed;
+  z-index: 1802;
+  // Without border-box the padding is added to the width, so the "360px" card
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)